frontend/src/ClassDetail.svelte

```diff
@@ -35,6 +35,18 @@
           .reduce((acc, val) => acc + val, 0).toFixed(2);
   }
 
+  function taskPoints(clazz, k) {
+  let assignmentPoints = 0;
+
+  for (let i = 0; i < clazz.students.length; i++) {
+    const student = clazz.students[i];
+      if (!isNaN(clazz.assignments[k].students[student.username].assigned_points)) {
```

Comment:
Sometimes we assign negative points. I think that it would be better to ignore these in the summary and treat any negative points as zero. You can add `Math.max(0, ... assigned_points)` to the running total.

Reply:
Fixed in the next commit called "Updated totalPointsPerAssignedTask".